README.md

```diff
@@ -42,3 +42,28 @@ or
 Auto Complete Script to help speed through the form journey.
 
 These scripts can be found in the docs directory.
+
+### Updating Tariff Code lists
```

Comment:
Could you add some info on why do we take this data from publicly available place?

Reply:
Done! :+1: